.dockerignore

```diff
@@ -0,0 +1,2 @@
+_build/
+deps/
```

Comment:
do you need to explicitly include any of the `.git` directories or does dockerignore file assume that one by default?
might want to include some of the files/extensions for helm too, unless you plan to put the charts in a different repo.